types/testutil/network/network.go

```diff
@@ -92,7 +91,7 @@ type Config struct {
 // DefaultConfig returns a sane default configuration suitable for nearly all
 // testing requirements.
 func DefaultConfig() Config {
```

Comment:
Every /x module imports `types/testutil` . In /x modules we call `network.DefaultConfig()`  that runs logic from a.go to break dependency on app/.

Reply:
> In /x modules we call network.DefaultConfig() that runs logic from a.go to break dependency on app/.

In x/ modules, inside their `client/testutil/suite.go` file, we should define a `func NewIntegrationTestSuite(cfg network.Config) *IntegrationTestSuite {}` function that takes a network config as argument. The tests are run inside `app/`, e.g. `app/modules_test.go`. Let's discuss this today on our 1:1 @cyberbono3